fullstop-whitelist/src/main/java/org/zalando/stups/fullstop/whitelist/WhitelistRulesEvaluator.java

```diff
@@ -95,25 +95,12 @@ public Boolean apply(RuleEntity ruleEntity, ViolationEntity violationEntity) {
     }
 
     private static Predicate<ViolationEntity> applicationIdIsEqual(String applicationId) {
-        return v -> {
-            if(v.getMetaInfo() instanceof Map) {
-                Map<String,String> map = (Map<String, String>) v.getMetaInfo();
-                return applicationId.equals(map.get("application_id"));
-            } else {
-                return false;
-            }
-        };
+        return v -> applicationId.equals(v.getApplication().getName());
```

Comment:
possible NullPointerException: there might be violations without application. With Optionals it could look like
```java
return v -> applicationId.equals(Optional.ofNullable(v.getApplication()).map(ApplicationEntity::getName).orElse(null));
```